code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -130,12 +130,17 @@
 	ENABLE_BITFIELD(reagents.reagents_holder_flags, NO_REACT)
 
 /obj/item/ammo_casing/shotgun/dart/bioterror
-	desc = "A shotgun dart filled with deadly toxins."
+	desc = "A shotgun dart filled with an obscene amount of lethal reagents. God help whoever is shot with this."
+	projectile_type = /obj/item/projectile/bullet/dart/piercing
```

Comment:
This was never piercing for a reason. 1 shooting cap/hos/engi hardsuits is already a huge problem with rockets. Why make the new meta this?

Reply:
Or you use an antitank pistol.

Reply:
Chem based combat is shit, the darts are not ment to be the best thing ever. This will shift the meta to just always buy darts + antitank pistol/CQC